shaky/src/main/java/com/linkedin/android/shaky/EmailShakeDelegate.java

```diff
@@ -32,6 +32,8 @@
 
     public EmailShakeDelegate(@NonNull String[] to) {
         this.to = to;
+        // Test changing the send icon by seeing if it's the navigation arrow
+        this.sendIcon = R.menu.shaky_change_icon_test;
```

Comment:
I don't think we need to put this test demo in the library module. Maybe in the sample app module or don't even put it.